langfuse/callback/langchain.py

```diff
@@ -1,4 +1,12 @@
 import logging
+
+log = logging.getLogger("langfuse")
+
+try:  # Test that langchain is installed before proceeding
+    import langchain
+except ImportError as e:
+    log.exception(f"Could not import langchain. Some functionality may be missing. {e}")
```

Comment:
May want to use `log.error` instead. The stack trace is ugly and not interesting. Langchain is just not installed and that's it.
May want to raise a clear exception here anyways, since the code will hit another import error a few lines down. The functionality is most definitely missing.

Reply:
Thanks!